recipe/meta.yaml

```diff
@@ -10,34 +10,49 @@ source:
   sha256: ec051cb568324de878828fae96379b71858933413e185148acb6c162851ab23c
 
 build:
-  number: 1
-  skip: true  # [py<38 or python_impl == 'pypy']
+  number: 0
+  # skip s390x: no libgdal
+  skip: true  # [py<39 or s390x]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
     - {{ compiler('c') }}
-    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
   host:
     - python
     - pip
     - setuptools
-    - cython
-    - libgdal-core
-    - versioneer
-    - tomli  # [py<311]
+    - cython >=0.29
+    # https://github.com/geopandas/pyogrio/blob/v0.10.0/setup.py#L24
+    - libgdal >=2.4.0
```

Comment:
Could you pin it like `libgdal {{ libgdal }}`? As it's in a generic cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L140-L141

Reply:
done